autodiscern/annotations.py

```diff
@@ -8,6 +6,7 @@
 
 
 def add_word_token_annotations(inputs: Dict[str, Dict]) -> Dict[str, Dict]:
+    from allennlp.data.tokenizers.word_tokenizer import WordTokenizer
```

Comment:
Given that allennlp is using spacy word tokenizer under the hood, I think maybe we can use directly spacy and remove the dependency for allennlp?

Reply:
We're not actually using word tokenizer in the models at this point, right? I moved the import into the function so we no longer have an official package dependency on allennlp, and I think that's good enough for now.

Reply:
For the neural-based models, we are using the tokenizer form BERT. However, for the baseline, are you using NLTK or SpaCy? This is also related to NER recognition in the text (I think we decided to use SpaCy for that). In both cases I am OK (my argument is to just use SpaCy given that it provides all text preprocessing that we need off-the-shelf).

Reply:
For the baseline models, I'm using nltk. The NER is from allennlp.